polylogue/doctor.py

```
@@ -355,13 +355,13 @@ def run_doctor(
     issues.extend(_credential_issues())
     issues.extend(_drive_failure_issues())
     try:
-        sqlite_notes = verify_sqlite_indexes(DB_PATH)
+        sqlite_notes = verify_sqlite_indexes(default_db_path())
         if sqlite_notes:
             counts["indexes"] = counts.get("indexes", 0) + len(sqlite_notes)
             for note in sqlite_notes:
-                issues.append(DoctorIssue("index", DB_PATH, note, "info"))
+                issues.append(DoctorIssue("index", default_db_path(), note, "info"))
     except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)